pages/index.tsx

```diff
@@ -255,27 +256,29 @@ const Status = (props: SubgraphIndexingStatus) => {
             {' ⛓️ '}
 
             <a
+              target="_blank"
               className="text-purple-600 hover:underline"
               href={`https://api.thegraph.com/explorer/graphql?query=${encodeURIComponent(
                 `{
-  subgraphLogs(
```

Comment:
shall we undo the indentation here? otherwise, if you go into the API site, it shows:
<img width="739" alt="image" src="https://user-images.githubusercontent.com/101405096/177227664-e3003d7e-4f9a-4a8e-afb3-83e4279483a0.png">

Reply:
that's why it was formatted weird. changed